newIDE/app/src/AnnouncementsFeed/AnnouncementsFeed.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import * as React from 'react';
+import AlertMessage from '../UI/AlertMessage';
+import { ColumnStackLayout } from '../UI/Layout';
+import PlaceholderError from '../UI/PlaceholderError';
+import PlaceholderLoader from '../UI/PlaceholderLoader';
+import RaisedButton from '../UI/RaisedButton';
+import { selectMessageByLocale } from '../Utils/i18n/MessageByLocale';
+import Window from '../Utils/Window';
+import { AnnouncementsFeedContext } from './AnnouncementsFeedContext';
+import Text from '../UI/Text';
+import { Line } from '../UI/Grid';
+import PreferencesContext from '../MainFrame/Preferences/PreferencesContext';
+
+type AnnouncementsFeedProps = {|
+  level?: 'urgent' | 'normal',
+  canClose?: boolean,
+  addMargins?: boolean,
+|};
+
+export const AnnouncementsFeed = ({
+  level,
+  canClose,
+  addMargins,
+}: AnnouncementsFeedProps) => {
+  const { announcements, error, fetchAnnouncements } = React.useContext(
+    AnnouncementsFeedContext
+  );
+  const { values, showAnnouncement } = React.useContext(PreferencesContext);
+
+  if (error) {
+    return (
+      <PlaceholderError onRetry={fetchAnnouncements}>
```

Comment:
Should it silently fail instead? It doesn't seem that important to take space in the IDE and show that there's an issue

Reply:
It does :) In the HomePage, if there are no announcements, it does not even try to show the AnnouncementsFeed at the top (and no loading too), so it's effectively "best effort".
Error and loading *are* displayed though in the community page.